src/components/ViewContainer.tsx

```diff
@@ -3,8 +3,8 @@ import { StyleSheet, View } from 'react-native';
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    alignItems: 'center',
+    //flex: 1,
+    //alignItems: 'center',
```

Comment:
Uncomment these and style your components accordingly (I made this change, sorry).